README.rst

```diff
@@ -139,15 +139,15 @@ The following code implements a simple dual moving average algorithm.
                long_mavg=long_mavg)
 
 
-You can then run this algorithm using the Zipline CLI. From the command
-line, run:
+You can then run this algorithm using the Zipline CLI; you'll need a `Quandl <https://www.quandl.com/>`__ API key for default data bundle.
```

Comment:
Might want to link instead to the "Getting an API key" section of Quandl's docs:

https://docs.quandl.com/docs#section-authentication

Reply:
"API key for default data bundle." -> "API key to download the default data bundle."